src/os/im/action/importactionmanager.js

```diff
@@ -399,7 +399,8 @@ os.im.action.ImportActionManager.prototype.processItemsProtected = function(
 os.im.action.ImportActionManager.prototype.processItems = function(entryType, opt_items, opt_unprocess) {
   var items = opt_items || this.getEntryItems(entryType);
   if (items && items.length > 0) {
-    this.processItemsProtected(entryType, items, opt_unprocess);
+    if (opt_unprocess) this.unprocessItems(entryType, items);
```

Comment:
This works, but goes against the concept of consolidating the events that processItemsProtected() does for you.
I was thinking more along the lines that processItemsProtected() in ImportActionManager/FeatureActionManager would:
1. Remember the last FA's they applied, and if not null, unprocess those first (and save the events into the config), then
2. Still do what it used to do (i.e. build the config for the events so it throws one event for each Feature)